src/fidesops/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -147,11 +147,15 @@ def create_privacy_request(
             failed.append(failure)
             continue
 
+        requested_at = privacy_request_data.requested_at
+        if requested_at is None:
+            requested_at = datetime.utcnow()
```

Comment:
This part should really be in the `PrivacyRequest.create(...)` method, I've only added it here for brevity